src/main/webapp/app/shared/sidebar/sidebar-accordion/sidebar-accordion.component.ts

```diff
@@ -8,6 +8,17 @@ import { SidebarCardDirective } from '../directive/sidebar-card.directive';
 import { ArtemisTranslatePipe } from 'app/shared/pipes/artemis-translate.pipe';
 import { SearchFilterPipe } from 'app/shared/pipes/search-filter.pipe';
 import { AccordionGroups, ChannelTypeIcons, CollapseState, SidebarCardElement, SidebarItemShowAlways, SidebarTypes } from 'app/shared/types/sidebar';
+import dayjs from 'dayjs/esm';
+import isoWeek from 'dayjs/plugin/isoWeek';
```

Comment:
please move this into dayjs.ts and also use the `.../esm/...` version for the import